components/event/EventCard.js

```diff
@@ -84,8 +84,8 @@ export default function EventCard({ manage, event, usernames }) {
                 <MdOutlineArrowRightAlt className="self-center hidden lg:block" />
                 <span>{endTime}</span>
               </p>
-              <ReactMarkdown className="text-sm text-primary-medium dark:text-primary-low-medium py-1 flex-wrap">
-                {event.description}
+              <ReactMarkdown className="text-sm text-primary-medium dark:text-primary-low-medium py-1 flex-wrap whitespace-pre-wrap ">
+                {event.description.replace(/\n/gi, '\n &nbsp;')}
```

Comment:
Duplicate logic in UserProfile component. Maybe it's better to move it to utils

Reply:
The markdown library we are using allows plugins, which looks like there is a plugin for line breaks from the user

- List of plugins https://github.com/remarkjs/remark/blob/main/doc/plugins.md#list-of-plugins
- Line break plugin https://github.com/remarkjs/remark-breaks

Reply:
Should I proceed with the previous logic i have used ?

Reply:
No. I will have a play with those plugins